src/get-started/install/linux.md

```diff
@@ -9,15 +9,32 @@ next:
 
 {% assign os = 'linux' -%}
 
-## System requirements
-
-To install and run Flutter,
-your development environment must meet these minimum requirements:
-
-- **Operating Systems**: Linux (64-bit)
-- **Disk Space**: 600 MB (does not include disk space for IDE/tools).
-- **Tools**: Flutter depends on these command-line tools being available
-  in your environment.
+| Main Section                             | Subsection                                                         |
+|------------------------------------------|--------------------------------------------------------------------|
+| 1. [Prerequisites](#prerequisites)       | [Operating Systems](#operating-systems)                            |
+|                                          | [Tools](#tools)                                                    |
+| 2. [Get the Flutter SDK](#get-sdk)       | [Understanding Linux Commands](#understanding-linux-commands)      |
+|                                          | [Install Flutter using snapd](#install-flutter-using-snapd)        |
+|                                          | [Manual Installation](#manual-installation)                        |
+|                                          | [Run flutter doctor](#run-flutter-doctor)                          |
+| 3. [Update your path](#update-your-path) | [Temporarily update your path](#temporarily-update-your-path)      |
+|                                          | [Permanently update your path](#permanently-update-your-path)      |
+|                                          | [Update the path directly](#update-the-path-directly)              |
+| 4. [Android setup](#android-setup)       | [Install Android Studio](#install-android-studio)                  |
+|                                          | [Set up your Android device](#setup-android-device)                |
+|                                          | [Set up the Android emulator](#setup-android-emulator)             |
+|                                          | [Agree to Android licenses](#agree-android-licenses)               |
+|                                          | [Install Google Chrome](#install-google-chrome)                    |
+| 5. [Linux setup](#linux-setup)           | [Additional Linux requirements](#additional-linux-requirements)    |
+|                                          | [Understanding Linux Commands](#understanding-linux-commands)      |
```

Comment:
**issue (blocking):** This section is generated automatically with the on-page Table of Contents. This should be removed.